src/server/pps/server/api_server.go

```diff
@@ -2448,7 +2454,7 @@ func (a *apiServer) initializePipelineInfo(txn *pps.CreatePipelineTransaction, o
 			OutputBranch:            request.OutputBranch,
 			Egress:                  request.Egress,
 			CreatedAt:               timestamppb.Now(),
-			UpdatedAt:               timestamppb.Now(),
+			UpdatedAt:               txn.When,
```

Comment:
Elsewhere we are using this field as the actual commit time.  I don't think we should deviate from that here.  If we wanted to track that, we could retain the transaction object and link it to the pipeline and commits.

Reply:
For consistency, I think you’re right — and although I think it is odd for a pipeline created at one moment to have its creation date recorded as potentially hours in the future, that could probably be addressed when more of the transaction system is addressed. Changed!

Reply:
I think it's fine.  Pachyderm doesn't start treating it as a pipeline until that moment.  It's just a note on someone's todo list before then.